src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -456,6 +476,78 @@ protected void setup()
         completeSetup();
     }
 
+    /**
+     * Checks if the data of the local system keyspaces need to be migrated to a different location.
+     *
+     * @throws IOException
+     */
+    private void migrateSystemDataIfNeeded() throws IOException
+    {
+        String importSystemDataFrom = System.getProperty("cassandra.importSystemDataFilesFrom");
```

Comment:
Probably we should comment the new property in [the documentation for cassandra-ev](https://github.com/apache/cassandra/blob/trunk/doc/source/configuration/cass_env_sh_file.rst) and in the [JVM options file](jvm-server.options).

Reply:
I added the property to the documentation. I am not sure if we should add it to the JVM options file as the property is a bit of a corner case. It is only usefull if you have used a dedicated directory for your system table and want to go back to using your data directories.